Gemfile

```diff
@@ -19,7 +19,8 @@ gem 'webpacker', '~> 3.5'
 group :development, :test do
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem "dotenv-rails"
-  gem 'rubocop', '~> 0.71.0', require: false
+  gem 'rubocop', '~> 0.72.0', require: false
```

Comment:
`0.73` is the last one apparently.
Can we jump to that one? :-)

Reply:
Sure!